database/src/main/postgres/runs/V1.5.11__additional_data_history.ddl

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE TABLE runs.additional_data_history
+(
+    id_additional_data_history  BIGINT NOT NULL DEFAULT global_id(),
+    fk_partitioning             BIGINT NOT NULL,
+    ad_name                     TEXT NOT NULL,
+    ad_value                    TEXT,
+    created_by_originally       TEXT NOT NULL,
+    created_at_originally       TIMESTAMP WITH TIME ZONE NOT NULL,
+    archived_by                 TEXT NOT NULL,
+    archived_at                 TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    CONSTRAINT additional_data_history_pk PRIMARY KEY (id_additional_data_history)
+);
+
+ALTER TABLE runs.additional_data_history
+    ADD CONSTRAINT ad_history_unq UNIQUE (fk_partitioning, ad_name, ad_value, created_by_originally, created_at_originally);
```

Comment:
> I can see that we have fk_partitioning but not really refferenced back to partitioning table, is there a reason for us to do that? Because I have noticed that in other tables as well.

what do you mean referenced back? Can you give me an example?

Reply:
> This is a weird constraint. I would like to discuss.

definitely. There is a reason for that, let's talk offline if still interested